src/IssueViz.Security/OpenInIDE/OpenInIDEResources.resx

```diff
@@ -132,6 +132,28 @@
   <data name="Factory_Succeeded" xml:space="preserve">
     <value>[Open in IDE] Request listener created successfully. Listening on port {0}.</value>
   </data>
+  <data name="RequestValidator_InvalidStateReason_NotInConnectedMode" xml:space="preserve">
+    <value>you must be in Connected Mode</value>
+  </data>
+  <data name="RequestValidator_InvalidStateReason_WrongOrganization" xml:space="preserve">
+    <value>you must connect to the right organization. Currently connected to: {0}</value>
+  </data>
+  <data name="RequestValidator_InvalidStateReason_WrongProject" xml:space="preserve">
+    <value>you must bind your solution to the right project. Currently bound to: {0}</value>
```

Comment:
"you must bind your solution to..." still sounds to me like an instruction to change the current solution.

Something like "The current solution is not bound to the requested Sonar project. Requested Sonar project: foo. Bound Sonar project: bar" would be less directive.

Also, "project" is used by MSBuild/MS devs and SonarSource to mean different things. It's always worth specifying which one we mean to avoid confusion.